app/models/external-storage-service.ts

```diff
@@ -11,10 +11,27 @@ export enum CredentialsFormat {
     ACCESS_SECRET_KEYS = 'access_secret_keys',
 }
 
+export enum TermsOfServiceCapabilities {
+    ADD_UPDATE_FILES = 'add_update_files',
+    ADD_UPDATE_FILES_PARTIAL = 'add_update_files_partial',
+    DELETE_FILES = 'delete_files',
+    DELETE_FILES_PARTIAL = 'delete_files_partial',
+    FORKING = 'forking',
+    FORKING_PARTIAL = 'forking_partial',
+    LOGS = 'logs',
+    LOGS_PARTIAL = 'logs_partial',
+    PERMISSIONS = 'permissions',
+    PERMISSIONS_PARTIAL = 'permissions_partial',
+    REGISTERING = 'registering',
+    REGISTERING_PARTIAL = 'registering_partial',
+    FILE_VERSIONS = 'file_versions',
+}
+
 export default class ExternalStorageServiceModel extends OsfModel {
     @attr('fixstring') name!: string;
     @attr('string') credentialsFormat!: CredentialsFormat;
     @attr('string') iconUrl!: string;
+    @attr('fixstringarray') termsOfService!: TermsOfServiceCapabilities[];
```

Comment:
Should this just be a stringarray?

Reply:
I had thought there was a `stringarray` transform, but it looks like it's either `array` or `fixstringarray`. Will update to just `array`